### PR TITLE
Update copy & reduce duplicate code

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?164895

### CODE REVIEW

**Comment by orangejenny** [comment]
@nickpell

**Comment by nickpell** [comment]
waiting on new build

File: corehq/apps/groups/templates/groups/group_members.html [hunk 1]
```diff
@@ -76,52 +76,43 @@
                 return;
             });
 
-            function success(name, id, gaEventLabel) {
+            function outcome(success, name, id, gaEventLabel) {
```
Comment (nickpell) [inline corehq/apps/groups/templates/groups/group_members.html]: I prefer ```isSuccess``` for booleans, but nonblocking

File: corehq/apps/groups/templates/groups/group_members.html [hunk 2]
```diff
@@ -76,52 +76,43 @@
...
-                    $('.alert').removeClass('alert-error alert-info').addClass('alert-success');
-                    $('.alert')[0].innerHTML = name + " was successfully saved";
-                    $('.alert').show();
-                    $(id).find(':button').enableButton();
+                    var alertClass, message;
+                    if (success) {
+                        alertClass = 'alert-success';
+                        message = 'Successfully saved ' + name.toLowerCase() + '.';
+                        unsavedChanges[name] = false;
+                    }
```
Comment (nickpell) [inline corehq/apps/groups/templates/groups/group_members.html]: I prefer ```} else {``` (disclaimer: I'm not super knowledgeable about JS conventions)

Comment (orangejenny) [inline corehq/apps/groups/templates/groups/group_members.html]: do we have conventions? (I support having conventions, just wondering what they are.)

Comment (nickpell) [inline corehq/apps/groups/templates/groups/group_members.html]: Not that I know of - github applies a fairly lenient linter to python code, but currently we don't have one for JS

Comment (benrudolph) [inline corehq/apps/groups/templates/groups/group_members.html]: i think dimagimon will yell at you for some things in JS like not having a semi colon. airbnb has a good style guide that I try to stick to: https://github.com/airbnb/javascript. (they use `} else {` )

Comment (nickpell) [inline corehq/apps/groups/templates/groups/group_members.html]: :+1:

**Pull request merged**